ravenframework/CodeInterfaceClasses/MooseBasedApp/MooseInputParser.py

```diff
@@ -255,6 +259,45 @@ def findInGetpot(trees, targetPath):
         break
   return found
 
+def findCsvInGetpot(trees):
+  """
+    Find the CSV output blocks in the input file
+    @ In, trees, list(TreeStructure.InputTrees), the tree(s) represent the input file
+    @ Out, list, list(TreeStructure.InputTrees), list of CSV blocks in the input file
```

Comment:
this looks like it's returning a bool, not a list, right?

Reply:
Good catch, I have updated the docstring.